app/views/govuk_component/analytics_meta_tags.raw.html.erb

```diff
@@ -77,6 +77,15 @@
   meta_tags["govuk:taxon-ids"] = taxon_ids.join(',') unless taxon_ids.empty?
   meta_tags["govuk:taxon-slug"] = taxon_slugs_without_theme.first unless taxon_slugs_without_theme.empty?
   meta_tags["govuk:taxon-slugs"] = taxon_slugs_without_theme.join(',') unless taxon_slugs_without_theme.empty?
+
+  def has_content_history(content_item)
+    details = content_item.fetch(:details, {})
+
+    (content_item[:public_updated_at] && details[:first_public_at] && content_item[:public_updated_at] != details[:first_public_at]) ||
```

Comment:
Are we sure that `:first_public_at` is the correct key? My understanding of this is that [it's an alias for `:first_published_at`](https://github.com/alphagov/whitehall/blob/master/app/presenters/publishing_api/payload_builder/first_public_at.rb#L5) and [not used consistently even in Whitehall where it originates](https://github.com/alphagov/whitehall/search?p=1&q=first_public_at&type=&utf8=%E2%9C%93). 

That said I've little knowledge of how government-frontend deals with these fields so I'm just raising this as a concern, if you are satisfied it's the right field then :+1: